src/ex3/commands/getCommand.js

```diff
@@ -0,0 +1,18 @@
+import { getData } from "../fileManeger.js";
+import chalk from "chalk";
+
+let tasksArr = [];
+
+export async function getTasksList() {
+    tasksArr = await getData()
+    
+    if (tasksArr.length === 0) {
+        console.log("There are no tasks in your list");
+        return;
+    }
+
+    for (let i = 0; i < tasksArr.length; i++)
```

Comment:
```suggestion
    tasksArr.forEach(task => {
      console.log(chalk[task.color](task.name));
    })
```
Or if you want to go wild:

```suggestion
    tasksArr.forEach(({ color, name}) => {
      console.log(chalk[color](name));
    })
```